app/main.py

```diff
@@ -2,0 +1,18 @@
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    masks_to_buy = 0
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except VaccineError:
+            return "All friends should be vaccinated"
```

Comment:
Code Style: Your error message should be more descriptive. It's better to specify the reason for the error. For example, you can change the error message to 'All friends should be vaccinated to visit the cafe'.